fastlane_core/lib/fastlane_core/ui/github_issue_inspector_reporter.rb

```diff
@@ -46,5 +51,10 @@ def print_issue_full(issue)
       puts "   #{Time.parse(issue.updated_at).to_pretty}"
       puts ""
     end
+
+    def print_open_link_hint(newline = false)
+      puts "" if newline
+      puts "🔗 You can ⌘ + double-click on links to open them directly in your browser." if /darwin/ =~ RUBY_PLATFORM
```

Comment:
If we need another change to re-trigger CI... can we replace the end check with `Helper.mac?`